projects/1.20-fabric/assets/xp-storage/xp_storage/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+  "item.xp_storage.crystallized_lapis": "晶化青金石",
+  "item.xp_storage.xp_book_upgrade_smithing_template": "锻造模板",
+  "item.xp_storage.smithing_template.xp_book_upgrade.additions_slot_description": "Add钻石、下界合金锭或下界之星",

```

```suggestion
  "item.xp_storage.smithing_template.xp_book_upgrade.additions_slot_description": "Add 钻石、下界合金锭或下界之星",

```
请问这句话是什么情况？

下面还有Add

---

忘记改了抱歉